composer.json

```diff
@@ -18,7 +18,7 @@
 	},
 	"require": {
 		"php": "^7.1",
-		"phpstan/phpstan": "^0.11.6"
+		"phpstan/phpstan": "^0.11.7"
```

Comment:
This change is bc break (phpstan 0.11.7 needs nette/utils>=3.0.1). So I'm thinking about a new major release or introduce a similar concept like `bleedingEdge.neon`

Reply:
I am not sure about this - is this bc break? Since you did not change anything to break. The new version will not simply be installed for somebody, who is using nette v2?

https://github.com/phpstan/phpstan/compare/0.11.6...0.11.7#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780

Reply:
This commit is problematic https://github.com/phpstan/phpstan/commit/080fd292b84c12ee0ae23717a50c786c33389236
It requires `nette/schema` which needs nette 3.x

Reply:
Yeah, I know. My link is broken, but I tried to show the same.

Regarding semver:
https://github.com/Ocramius/ProxyManager/issues/455#issuecomment-474503368